features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Documents/Listing.jsx

```diff
@@ -105,29 +108,48 @@ class Listing extends React.Component {
      Get the document list attached to current API and set it to the state
      */
     getDocumentsList() {
-        const api = new API();
+        const { api } = this.props;
         const { intl } = this.props;
```

Comment:
Shall we merge these 2 lines

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6759/commits/4f501bf1f995ac5a20a84923f720b71550950f0a